composer.json

```diff
@@ -99,15 +99,15 @@
         "drupal/webform": "^5.4",
         "drupal/xmlsitemap": "~1.0@alpha",
         "ext-imagick": "*",
-        "su-sws/jumpstart_ui": "^8.1.0",
-        "su-sws/nobots": "^8.1.0",
-        "su-sws/react_paragraphs": "^8.1.0",
-        "su-sws/stanford_basic": "^8.4.0",
-        "su-sws/stanford_date_formats": "^8.1.0",
-        "su-sws/stanford_fields": "^8.1.0",
-        "su-sws/stanford_image_styles": "^8.1.0",
-        "su-sws/stanford_ssp": "^8.1.1",
-        "su-sws/stanford_text_editor": "^8.1.0"
+        "su-sws/jumpstart_ui": "dev-8.x-1.x@dev",
+        "su-sws/nobots": "dev-8.x-2.x@dev",
+        "su-sws/react_paragraphs": "dev-8.x-1.x@dev",
+        "su-sws/stanford_basic": "dev-8.x-4.x@dev",
+        "su-sws/stanford_date_formats": "dev-8.x-1.x@dev",
+        "su-sws/stanford_fields": "dev-8.x-1.x@dev",
+        "su-sws/stanford_image_styles": "dev-8.x-1.x@dev",
+        "su-sws/stanford_ssp": "dev-8.x-1.x@dev",
+        "su-sws/stanford_text_editor": "dev-8.x-1.x@dev"
```

Comment:
Is the `@dev` required when you are on the branch already? FWIK the `@` flag allows for an override of the minimum-stable version. Not sure if that is needed when using the branch naming convention.

Reply:
When I was testing, I ran into minimum-stability problems, which the `@dev` seemed to solve.  We have minimum-stability set to `dev` above, and we also have `prefer-stable` set to `true`.  However, we are always including this profile as a dependency in another composer file, where the minimum stability may be different, and `prefer-stable` may be getting in the way.  I'll test this a little more and see if it's essential or not.

Reply:
I tested this without the `@dev` suffix, and it appears to work correctly, so I've gone ahead and removed them.